Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -67,14 +68,14 @@ private class TopSitesLayout: UICollectionViewLayout {
     private let ToolbarHeight: CGFloat = 44
     private let StatusBarHeight: CGFloat = 20
     private let RowHeight: CGFloat = 58
-    private let AspectRatio: CGFloat = 0.8
+    private let AspectRatio: CGFloat = 1.25
```

Comment:
Heh, yeah, I changed it here because "aspect ratio" is supposed to be width:height, and we had it backwards. I'll add a comment just to be extra clear.